content/tutorial/index.md

```diff
@@ -15,17 +16,19 @@ If you are someone who prefers to stare at the code till it explains itself, you
   - [Passing Props to Components](#passing-props-to-components)
   - [Add JavaScript to Components](#add-javascript-to-components)
 - [Dynamic Page Generation](#dynamic-page-generation)
+
 ---
 
 ## What is Abell?
 
 Abell is a JavaScript-based Static-Site-Generator to help you build fast and scalable static websites. It is similar to tools like Jekyll, Eleventy, Hugo, and Hexo.
 
-Abell files are like HTML files except you can write JavaScript in double curly brackets which is executed to build you a static page.
```

Comment:
"build you a static page" is correct I believe. It is building a static page for you right?

Reply:
Reverted back to original